package.json

```
@@ -24,6 +24,7 @@
   "scripts": {
     "build": "vite build",
```

Should this be part of the existing `build` script?

```suggestion
    "build": "vite build && npm run copy-dts",
```